sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -377,12 +378,14 @@ def __init__(
       self,
       fallback_coder_impl,
       requires_deterministic_step_label=None,
-      force_use_dill=False):
+      force_use_dill=False,
+      skip_use_relative_filepaths=False):
```

Comment:
in general, i try to avoid double-negations. would it make sense to use

`use_relative_filepaths=True` instead?